bin/_mocha

```diff
@@ -68,8 +68,8 @@ program
   .option('-S, --sort', "sort test files")
   .option('-b, --bail', "bail after first test failure")
   .option('-d, --debug', "enable node's debugger, synonym for node --debug")
-  .option('-g, --grep <pattern>', 'only run tests matching <pattern>')
-  .option('-f, --fgrep <string>', 'only run tests containing <string>')
+  .option('-g, --grep <pattern>', 'only run tests having full title matching <pattern>')
+  .option('-f, --fgrep <string>', 'only run tests having full title containing <string>')
```

Comment:
I agree that it’s tricky — as I mentioned on the issue, I was quite surprised myself to have found how `--grep` actually works. But thinking about it a bit more, I realized that this is actually a good choice. 👷

If you have a clearer way to word this, I’m open to discuss it. 🤓